### ISSUE TITLE
Write access to S3 bucket?

### ISSUE THREAD

**Issue by jjmontgo** [carltonlee1/aws-wafv2-tf-demo#1]
Doesn't the firehose role need to be given write access to the S3 bucket?

**Comment by carltonlee1** [comment]
Yes, it does. At a minimum, I think it needs `s3:PutObject`. I'll probably fix at some point. And sorry about that. This was a quick write up loosely based on something I had done in the past.
